fuzzers/forkserver_simple/src/main.rs

```diff
@@ -145,13 +145,11 @@ pub fn main() {
         None => [].to_vec(),
     };
 
-    let mut tokens = Tokens::new();
-    let forkserver = ForkserverExecutorBuilder::new()
+    let forkserver = ForkserverExecutor::builder()
         .program(res.value_of("executable").unwrap().to_string())
         .args(&args)
         .debug_child(debug_child)
         .shmem_provider(&mut shmem_provider)
-        .autodict_tokens(&mut tokens)
```

Comment:
no more tokens?

Reply:
btw why not `auto_tokens`?

Reply:
the instrumentation side is 100% afl++, so I decided to use the original name

Reply:
Let's keep it consistent in the project(?)